packages/entity/src/AuthorizationResultBasedEntityMutator.ts

```diff
@@ -282,9 +283,12 @@ export class AuthorizationResultBasedCreateMutator<
 
     const insertResult = await this.databaseAdapter.insertAsync(queryContext, this.fieldsForEntity);
 
-    queryContext.appendPostCommitInvalidationCallback(
-      entityLoader.utils.invalidateFieldsAsync.bind(entityLoader, insertResult),
-    );
+    queryContext.appendPostCommitInvalidationCallback(async () => {
+      entityLoader.utils.invalidateFieldsForTransaction(queryContext, insertResult);
+      await entityLoader.utils.invalidateFieldsAsync(insertResult);
```

Comment:
if insertResult is already being invalidated for all txns in the tree, what is the purpose of `invalidateFieldsAsync`?

Reply:
I'll add a comment